bundles/org.openhab.binding.ecotouch/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,1005 @@
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+	<channel-type id="temperature_flow">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature Flow</label>
+		<description></description>
+		<category>Temperature</category>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+	<channel-type id="temperature_condensation" advanced="true">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature Condensation</label>
+		<description></description>
+		<category>Temperature</category>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+	<channel-type id="pressure_condensation" advanced="true">
+		<item-type>Number:Pressure</item-type>
+		<label>Pressure Condensation</label>
+		<description></description>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+	<channel-type id="temperature_storage" advanced="true">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature Storage</label>
+		<description></description>
+		<category>Temperature</category>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+	<channel-type id="temperature_room">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature Room</label>
+		<description></description>
+		<category>Temperature</category>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+	<channel-type id="temperature_room_1h">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature Room 1h Average</label>
+		<description></description>
+		<category>Temperature</category>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+	<channel-type id="temperature_water">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature Water</label>
+		<description></description>
+		<category>Temperature</category>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+	<channel-type id="temperature_pool">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature Pool</label>
+		<description></description>
+		<category>Temperature</category>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+	<channel-type id="temperature_solar">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature Solar</label>
+		<description></description>
+		<category>Temperature</category>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+	<channel-type id="temperature_solar_flow" advanced="true">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature Solar Flow</label>
+		<description></description>
+		<category>Temperature</category>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+	<channel-type id="position_expansion_valve" advanced="true">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Position Expansion Valve</label>
+		<description></description>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+	<channel-type id="power_compressor">
```

Comment:
```suggestion
	<channel-type id="power_compressor" advanced="true">
```
Can you mark any channels which wont be used often as advanced? This way people can use the 'select all' button and only get the most useful channels without needing to go through a big list, or worse selecting them all and have a large amount of channels then using persistence and using system resources. To see the advanced channels you just tick 'show advanced' when linking channels.

Reply:
I'm unsure about the channels which should go into advanced. I tried to mark some as advanced, but that depends on what the users want to do. I'll try to put many more channels into advanced and only keep a dozen or so.

Reply:
Sounds good as only you and people that use the binding can make that call. I usually ask two questions to myself when deciding

1. Can it be useful in a rule?
2. What are the channels that most users will want to add?

if it’s just a nice to have channel but not that useful to actually trigger an automation I tend to want it hidden under advanced. You can always hit the add all button and then tick show advanced and add a few more.